docs/reference/setup/install/package-security.asciidoc

```diff
@@ -48,24 +48,19 @@ the new node for the first time.
 +
 [source, sh]
 ----
-bin/elasticsearch-create-enrollment-token -s node
+/usr/share/elasticsearch/bin/elasticsearch-create-enrollment-token -s node
 ----
 
 . Copy the enrollment token, which is output to your terminal.
 
-. On your new {es} node, pass the enrollment token as a parameter to the 
+. On your new {es} node, pass the enrollment token as a parameter to the
 `elasticsearch-reconfigure-node` tool:
 +
 [source, sh]
 ----
-bin/elasticsearch-reconfigure-node --enrollment-token <enrollment-token>
+/usr/share/elasticsearch/bin/elasticsearch-reconfigure-node --enrollment-token <enrollment-token>
 ----
 +
 {es} is now configured to join the existing cluster.
 
-. Start your new node.
-+
-[source, sh]
-----
-bin/elasticsearch
-----
+. Start your new node as described in the "Running Elasticsearch with `systemd`" section below
```

Comment:
Is there any way to make this an actual link @jrodewig ?  :pray:

Reply:
Added in https://github.com/elastic/elasticsearch/commit/401ad730d0c20fe0511e081543142ecb441a1a32